docs/tempo/website/troubleshooting/_index.md

```diff
@@ -4,10 +4,15 @@ weight: 470
 ---
 
 # Troubleshooting missing traces
-This topic helps with day zero operational issues that may come up when getting started with Tempo. It walks through debugging each part of the ingestion and query pipeline to drill down and diagnose issues. 
+This topic helps with day zero operational issues that may come up when getting started with Tempo. It walks through debugging each part of the ingestion and query pipeline to drill down and diagnose issues.
```

Comment:
This page is getting a bit too long and scary, can we also split out the individual problems into smaller pages?